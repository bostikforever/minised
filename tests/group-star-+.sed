s/\(ab\)\+/_\1_/
