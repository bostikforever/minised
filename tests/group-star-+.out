x_ab_y
_ab_
c
